customers/services/profile.py

```diff
@@ -367,7 +374,7 @@ def create_profile(
         response = self.query(query=mutation, variables=variables)
         global_id = response.get("createProfile", {}).get("profile", {}).get("id")
         profile_id = UUID(from_global_id(global_id))
-        profile = CustomerProfile.objects.create(id=profile_id)
+        profile = CustomerProfile.objects.get(id=profile_id)
```

Comment:
What's this change?

Reply:
Good catch! It's an accident! I used the part of the script to import local data